cmd/authd-oidc/daemon/daemon_test.go

```diff
@@ -116,9 +116,9 @@ func TestAppRunFailsOnComponentsCreationAndQuit(t *testing.T) {
 		dataDirBehavior int
 		configBehavior  int
 	}{
-		"Error on existing data dir being a file":    {dataDirBehavior: dirIsFile},
-		"Error on data dir missing parent directory": {dataDirBehavior: noParentDir},
-		"Error on wrong permission on data dir":      {dataDirBehavior: wrongPermission},
+		"Error_on_existing_data_dir_being_a_file":    {dataDirBehavior: dirIsFile},
```

Comment:
The whole purpose of using capitalized letters for the test cases was because they were supposed to be "English sentences". Now that we are using a more "computer-like" format, maybe we can also drop the uppercased first letter rule?